content/integrations/windows_event_log.md

```diff
@@ -0,0 +1,84 @@
+---
+integration_title: Windows Event Log
+name: windows_event_log
+kind: integration
+description: "Collect Windows Events as logs in Datadog"
+is_public: true
+public_title: Datadog-Windows Event Integration
+short_description: "Collect Windows Events as logs in Datadog"
+categories:
+- log collection
+- os & system
+doc_link: https://docs.datadoghq.com/integrations/windows_event_log/
+supported_os:
+- windows
+ddtype: check
+---
+
+## Overview
+
+Collect Windows Events as logs in Datadog in order to:
+
+* Track system and application events in Datadog.
+* Correlate Windows system and application events with the rest of your application.
+* Track Security events and troubleshoot potential breach.
```

Comment:
Should "Security" be capitalized? If it's the official name of a type of event, then yes, but if it's just general security-related events then it should be lowercase.

Also, for grammar, it should be "troubleshoot potential breaches."